packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -43,6 +44,8 @@ export interface AlertProps extends Omit<React.HTMLProps<HTMLDivElement>, 'actio
   onTimeout?: () => void;
   /** Truncate title to number of lines */
   truncateTitle?: number;
```

Comment:
good catch.  That is a regression.  I am looking at it now.

Reply:
@mcoker when we apply the tooltip, it looks like the maxLines value we set is not applied.  Do you have an ideas why that is?  If i se the `truncateTile` to a number greater than the number of line of text, that work because we do not have a tooltip.  But if we have a tooltip, it always truncates to 1 line.

Reply:
Looks like there is some problem here https://github.com/patternfly/patternfly-react/blob/def3bc158ccb6cee89383783d6ed1eca3a9e16f3/packages/react-core/src/components/Alert/Alert.tsx#L85

This should create an inline style that defines `--pf-c-alert__title--max-lines` with the value of the truncate prop. Should look like this

<img width="240" alt="Screen Shot 2020-10-23 at 12 37 48 PM" src="https://user-images.githubusercontent.com/35148959/97036121-f5b5aa80-152c-11eb-8737-3fd41802885a.png">